agentops/llms/anthropic.py

```diff
@@ -31,27 +31,27 @@ def handle_response(
         from anthropic.resources import AsyncMessages
         from anthropic.types import Message
 
-        self.llm_event = LLMEvent(init_timestamp=init_timestamp, params=kwargs)
+        llm_event = LLMEvent(init_timestamp=init_timestamp, params=kwargs)
```

Comment:
I haven't run this yet, but why are we moving away from class variables? My thought was that we were doing this because if you declare it globally and you have multiple threads, they will interfere

Reply:
My guy this was the bug fix